### ISSUE TITLE
Halo mcc freezes after logging into my account

### ISSUE THREAD

**Issue by warriormaster12** [ValveSoftware/steam-runtime#360]
#### Your system information

* Steam Runtime Version: latest stable
* Distribution (e.g. Ubuntu 18.04): KDE Neon 20.04 lts using mainline kernel 5.10.10. Tested Ubuntu provided kernels, just in case.
* Link to your full system information (Help -> System Information) in a [Gist](https://gist.github.com/):   
[systemInfo.txt](https://github.com/ValveSoftware/steam-runtime/files/5898747/systemInfo.txt)

* Have you checked for system updates?: Yes
* Are you using the Steam Linux Runtime compatibility tool?: Yes

#### Please describe your issue in as much detail as possible:
Basically, when I try to log into my Xbox live account the game freezes and I have to force quit it. If I want to play the game, I have to use a version of Proton that doesn't use pressure-vessel. 

#### Steps for reproducing this issue:

1. Select version of proton that uses pressure vessel
2. Launch the game and log into your Xbox live account.

[steam-976730.log](https://github.com/ValveSoftware/steam-runtime/files/5898748/steam-976730.log)

**Comment by RyuzakiKK** [comment]
I just tried to reproduce the issue but I was able to successfully log in without any crashes.

@warriormaster12 Can you please try it again? Maybe one of the recent Proton or SLR Soldier updates fixed this issue.

**Comment by kisak-valve** [comment]
Closing pending feedback.
